src/EditorFeatures/TestUtilities/Completion/AbstractCompletionProviderTests.cs

```diff
@@ -149,12 +150,14 @@ internal static CompletionHelper GetCompletionHelper(Document document)
             }
         }
 
+        protected virtual ExportProvider ExportProvider => null;
```

Comment:
@dpoeschl With change, it would be easier to create tests for experiment service. You probably can undo related changes you made in the test harness.